js/core/dom_component.js

```diff
@@ -5,6 +5,7 @@ import windowResizeCallbacks from '../core/utils/resize_callbacks';
 import { Component } from './component';
 import { TemplateManager } from './template_manager';
 import { attachInstanceToElement, getInstanceByElement } from './utils/public_component';
+import { addShadowDOMStyles } from './utils/shadow-dom';
```

Comment:
We use CamelCase for abbreviations
```suggestion
import { addShadowDomStyles } from './utils/shadow-dom';
```